components/brave_wallet/browser/solana_instruction_builder.cc

```diff
@@ -0,0 +1,126 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/browser/solana_instruction_builder.h"
+
+#include <type_traits>
+
+#include "brave/components/brave_wallet/browser/brave_wallet_constants.h"
+#include "brave/components/brave_wallet/browser/solana_account_meta.h"
+#include "brave/components/brave_wallet/browser/solana_instruction.h"
+#include "brave/components/brave_wallet/common/brave_wallet_types.h"
+#include "build/build_config.h"
+
+namespace {
+
+template <typename T>
+void UintToLEBytes(T val, std::vector<uint8_t>* bytes) {
```

Comment:
It's better to put comment saying this is doing `bincode::serialize` to encode number

Reply:
Addressed in https://github.com/brave/brave-core/pull/12170/commits/6fb9d4d34cee63614dc57f708700f8b36365379e